source/blog/2016-12-14-size-and-change.html.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Size and Change
+date: 2016-12-14 15:30 UTC
+author: dneary
+tags: community, development, scale
+published: true
+comments: true
+---
+![Apple and Orange](blog/Apple_and_Orange.jpg) I have been thinking about how size can affect culture and adaptability of groups recently. The topic once again came up today in a talk about what makes a healthy community. The answer to that will depend on the community's size and maturity. An open source project, in the words of one participant in this conversation, should have "the minimum level of structure to allow it to function effectively." I agree&mdash;just enough is the right amount. This article contains some ponderings on the relationship between size and communities, and some conclusions we can take from that.
+
+READMORE
+
+## Size and Culture
+
+I have been thinking of this because of a few converging issues&mdash;one project I work on, which started with four people, now has more activity than we can handle, and we are thinking of how to grow the group of (OpenStack style) committers. I have been helping another project figure out how to create a governance structure to allow them to grow
+effectively beyond their current size, where they have about 10 active companies and about 150 active developers. And the company I work for has been going through some growing pains as we go past 10,000 people in size.
+
+The work of Robin Dunbar comes to mind&mdash;he of [Dunbar's Number](https://en.wikipedia.org/wiki/Dunbar's_number). In [How Many Friends Does One Person Need?](http://www.hup.harvard.edu/catalog.php?isbn=9780674057166), he examines the nature of communities of different sizes. He finds a number of group sizes that
+recur often, and their characteristics change at certain size limits: The specific sizes he points to are 5, 15, 50, 150, 500, and 5000&mdash;he
+suggests that there might be something special about this scaling factor
+of three.  But he does not talk a lot about evolution&mdash;how group dynamics
+change as the groups grow. A few examples occur to me from my own
+experiences:
+
+* One small company I worked for used to have a monthly dinner with
+all staff and their partners&mdash;possible when there were five or six employees
+and 8 or 10 people at dinner, but that gets more difficult as you get
+bigger.
+
+* Another company, we were between 15 and 30 employees while I
+worked there&mdash;every morning, everyone, when arriving, used to say good
+morning to everyone else in the company. At some point (around 20
+people), people stopped doing that, and just said hello to the people
+they met on their way to their desk, and the people in their immediate
+office.
+
+* As mailing lists grow, the peer pressure to respond personally to
+any individual message diminishes. At a certain size, it becomes much
+more difficult to evaluate the authority level of a person posting on
+the list, and the social ties weaken between list participants. On one
```

Comment:
Perhaps "On one list I participated in" to make it more concrete (this has happened to me several times)